include/envoy/upstream/cluster_manager.h

```diff
@@ -134,6 +134,13 @@ class ClusterManager {
    *                     or "static" if CDS is not in use.
    */
   virtual const std::string versionInfo() const PURE;
+
+  /**
+   * Return the local cluster name, if it was configured.
+   *
+   * @return std::string the local cluster name, or "" if no local cluster was configured.
+   */
+  virtual const std::string localClusterName() const PURE;
```

Comment:
I think `const std::string` doesn't do anything here, you probably either want `std::string` or `const std::string&`.